### ISSUE TITLE
relations between GeochronologicEra and sub - eras

### ISSUE THREAD

**Issue by smrgeoinfo** [CGI-IUGS/timescale-ont#7]
gts currently used time:intervalContains to relate gts:GeochronologicEras to suberas, e.g. 
`gts:Period time:intervalContains some gts:Epoch`

this is problematic because the definition of time:intervalContains is "If a proper interval T1 is intervalContains another proper interval T2, then the beginning of T1 is before the beginning of T2, and the end of T1 is after the end of T2."

The Cretaceous period has only two Epochs (Lower and Upper); one starts and one finishes, but neither is contained, so the restriction is violated.   It seems that using thors:member would solve the problem-- it has time:intervalContains, time:intervalFinishedBy, and time:intervalStartedBy as sub properties. 
so `gts:Period thors:member some gts:Epoch` would work without problem.

**Comment by dr-shorthair** [comment]
@smrgeoinfo yes, you are correct that the 'existential constraints' involving `time:intervalContains` are problematic. 
I think you can see what I was trying to do, but where there are only two intervals of the next rank down, then there will be none that can be related using the `time:intervalContains` predicate, since it is disjoint with `time:intervalStartedBy` and `time:intervalFinishedBy`. 

`gts:Period time:intervalContains only gts:Epoch` is also incorrect, since a `Period` may also contain `Ages`. 

`gts:Period thors:member some gts:Epoch` (etc) (as you suggest) looks OK, but probably should be accompanied by 
```turtle
time:intervalContains rdfs:subPropertyOf thors:member .
time:intervalStartedBy rdfs:subPropertyOf thors:member .
time:intervalFinishedBy rdfs:subPropertyOf thors:member .
```
